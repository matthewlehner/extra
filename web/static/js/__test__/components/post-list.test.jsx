// @flow

import React from "react";
import renderer from "react-test-renderer";

import PostList from "components/post-list";

describe("PostList component", () => {
  it("renders a list of posts", () => {
    const props = {
      posts: [{
        id: "1",
        body: "hey buddy."
      }]
    };

    const tree = renderer.create(<PostList {...props} />);

    expect(tree).toMatchSnapshot();
  });
});
