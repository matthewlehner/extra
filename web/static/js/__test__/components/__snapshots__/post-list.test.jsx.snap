// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostList component renders a list of posts 1`] = `
<section
  className="posts-collection"
>
  <header>
    <h2>
      Posts
    </h2>
  </header>
  <ul
    className="posts-list"
  >
    <li
      className="post-preview"
    >
      <p>
        hey buddy.
      </p>
    </li>
  </ul>
</section>
`;
